hack/jenkins/test-dockerized.sh

```diff
@@ -47,14 +36,10 @@ pushd "./hack/tools" >/dev/null
   GO111MODULE=on go install gotest.tools/gotestsum
 popd >/dev/null
 
-# Enable the Go race detector.
-export KUBE_RACE=-race
 # Disable coverage report
 export KUBE_COVER="n"
 # Set artifacts directory
 export ARTIFACTS=${ARTIFACTS:-"${WORKSPACE}/artifacts"}
-# Produce a JUnit-style XML test report
-export KUBE_JUNIT_REPORT_DIR="${ARTIFACTS}"
```

Comment:
the scripts all default this to match artifacts unless otherwise set. simplify this script a bit more.

ideally test-dockerized should go away, but integration job still kinda needs it.